src/gen/conditions.rs

```diff
@@ -217,6 +217,15 @@ pub fn parse_args(
                 }
             }
         }
+        256..=511 => {
```

Comment:
Integer representations are not unique (for example, 0x0100 and 0x000100 are both 256). Does `sanitize_uint` require canonical representation? I'm just curious if we can use `0x000100` here. Either way is fine, just want to make sure there is intent.

Reply:
I don't think `sanitize_uint` is involved in the parsing of the condition codes, but codes with redundant leading zeroes are not interpreted as the codes in canonical form. See my other comment